cmd/minishift/cmd/util/util.go

```diff
@@ -61,6 +64,18 @@ func ExitIfNotRunning(driver drivers.Driver, machineName string) {
 	}
 }
 
+func GetVmStatus(profileName string) string {
```

Comment:
👍

Reply:
but rename consistently with:

```
func VMExists(
```

So use `GetVMStatus` instead

Reply:
@gbraad `GetVmStatus`  follows correct camel case. The others functions do not use correct camel case. So I do not want to change this.

Reply:
Consistency is more important than personal opinion